File: InteractiveLive/Android/VideoOne/component/ToolKit/src/main/java/com/vertcdemo/core/entity/LoginInfo.java
// Copyright (c) 2023 BytePlus Pte. Ltd.
// SPDX-License-Identifier: Apache-2.0

package com.vertcdemo.core.entity;

import androidx.annotation.NonNull;

import com.google.gson.annotations.SerializedName;

public class LoginInfo {
    @SerializedName("user_id")
    public String userId;
    @SerializedName("user_name")
    public String userName;
    @SerializedName("login_token")
    public String loginToken;

    @NonNull
    @Override
    public String toString() {
        return "LoginInfo{" +
                "userId='" + userId + '\'' +
                ", userName='" + userName + '\'' +
                ", loginToken='" + loginToken + '\'' +
                '}';
    }
}

File: Client/Android/component/solution-base/src/main/kotlin/com/vertcdemo/core/entity/LoginInfo.kt
// Copyright (c) 2023 BytePlus Pte. Ltd.
// SPDX-License-Identifier: Apache-2.0
package com.vertcdemo.core.entity

import com.google.gson.annotations.SerializedName

class LoginInfo {
    @JvmField
    @SerializedName("user_id")
    var userId: String? = null

    @JvmField
    @SerializedName("user_name")
    var userName: String? = null

    @JvmField
    @SerializedName("login_token")
    var loginToken: String? = null

    override fun toString() =
        "LoginInfo{userId='${userId}', userName='$userName', loginToken='$loginToken'}"
}
